uniflow/flow/extract/extract_html_flow.py

```diff
@@ -35,3 +36,14 @@ def run(self, nodes: Sequence[Node]) -> Sequence[Node]:
         nodes = self._process_html_op(nodes)
         nodes = self._split_op(nodes)
         return nodes
+
+
+class ExtractS3HTMLFlow(ExtractHTMLFlow):
+    """Extract HTML Flow Class."""
+
+    def __init__(self, splitter: str = PARAGRAPH_SPLITTER) -> None:
+        """Extract HTML Flow Constructor."""
+        super().__init__()
+        self._extract_html_op = ExtractS3Op(name="extract_s3_html_op")
```

Comment:
Through pair programming, we found that `ExtractHTMLOp` also have a `parse_html` method to use beautiful soup to parse the html.

We propose a quick and dirty fix for now to write another `ExtractHTMLS3Op` to include this parsing step as well.